boat-scaffold/src/main/templates/boat-angular-mock/package.mustache

```diff
@@ -6,6 +6,12 @@
     {{#specVersion}}
     "specVersion": "{{{specVersion}}}"
     {{/specVersion}}
+    {{#artifactId}}
+    "artifactId": "{{{artifactId}}}"
+    {{/artifactId}}
+    {{#groupId}}
+    "groupId": "{{{groupId}}}"
+    {{/groupId}}
```

Comment:
I think it would be better to use `specArtifactId` and `specGroupId` for consistency with the existing `specVersion` property.

Reply:
https://github.com/Backbase/backbase-openapi-tools/pull/490/commits/a55069047a37939e34cd8cb2459f49abdc852da1

Reply:
Updated, I notice some tests fail though (`BoatMarinaTest.java`) that seem to be unrelated to this PR change:

```
 [INFO] [c.b.o.c.m.BoatMarinaTest] Generate docs for: /home/runner/work/backbase-openapi-tools/backbase-openapi-tools/boat-scaffold/target/test-classes/openapi-with-examples/openapi-with-json.yaml
[INFO] [o.o.c.DefaultGenerator] Generating with dryRun=false
[INFO] [o.o.c.DefaultGenerator] OpenAPI Generator: boat-marina (documentation)
[INFO] [o.o.c.DefaultGenerator] Generator 'boat-marina' is considered stable.
Error:  [o.o.c.l.StaticHtml2Generator] OpenAPI object description is empty [Simple API overview]
```

Looking at ths YAML file it's [missing a description property,](https://github.com/Backbase/backbase-openapi-tools/blob/0cf2f2162586fa65a524d5430c48f4be22c21660/boat-scaffold/src/test/resources/openapi-with-examples/openapi-with-json.yaml#L4) which causes the test [to fail on this assert](https://github.com/Backbase/backbase-openapi-tools/blob/0cf2f2162586fa65a524d5430c48f4be22c21660/boat-scaffold/src/test/java/com/backbase/oss/codegen/marina/BoatMarinaTest.java#L45).

I could add the missing description value to the YAML file, but I think the point here is that `marina` generator should be setting this default description (_No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)_) in the YAML file?

Reply:
It was failing for me in a different place 🤔